hack/tools/release/weekly/main.go

```diff
@@ -169,9 +169,24 @@ func run() int {
 		merges[key] = append(merges[key], formatMerge(body, prNumber))
 	}
 
+	// fetch the current branch
+	out, err = exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD").CombinedOutput()
+	if err != nil {
+		fmt.Println("Error")
+		fmt.Println(err)
+		fmt.Println(string(out))
```

Comment:
is this a debug print statement?  if there is an error in attempting to assign the value and we are about to `return 1`, why print this?

Reply:
Similar comment as above.